xml/schema/Core/PrevNextCache.xml

```diff
@@ -72,4 +72,17 @@
     <fieldName>entity_table</fieldName>
     <fieldName>is_selected</fieldName>
   </index>
+  <field>
+    <name>expiry_date</name>
+    <title>Prev Next Expiry Date</title>
+    <type>timestamp</type>
+    <default>current_timestamp()</default>
```

Comment:
:+1: for `timestamp`.

The name `expiry_date` stuck out as me as unfamiliar and had me thinking, "we should be consistent!" Well, it turns out... this is consistent! Or, more strictly, every field involving the end times uses a different name (`expired_date`, `expiry_date`, `expires_on`,  `expiration_time`, `hush_until`). One of them is using `expiry_date`, so this is more consistent. :)

I guess if you wanted to be *really* consistent, you'd have to find an entirely new name for the field. 😜 One unclaimed phrase is `kick_the_cache_bucket_at`.